### ISSUE TITLE
Documentation for `list` command not on `list` url

### ISSUE THREAD

**Issue by sth** [yarnpkg/website#355]
The documentation for `list` is still found on https://yarnpkg.com/en/docs/cli/ls, not https://yarnpkg.com/en/docs/cli/list . It should be renamed and a redirect from `ls` to `list` should de added.

Yarn itself already points to https://yarnpkg.com/en/docs/cli/list in its help message:
```
$ yarn ls 
yarn ls v0.19.1
error Did you mean `yarn list`?
info Visit https://yarnpkg.com/en/docs/cli/list for documentation about this command.
```
Currently this URL shows a 404.

**Comment by grant** [comment]
Fixes part of the issue: https://github.com/yarnpkg/website/pull/356

**Comment by Haroenv** [comment]
closed of #377
